Control/test/public/page-new-environment-mocha.js

```diff
@@ -133,40 +133,13 @@ describe('`pageNewEnvironment` test-suite', async () => {
     assert.deepStrictEqual(selectedWorkflow.classList, {0: 'w-90', 1: 'menu-item', 2: 'w-wrapped', 3: 'selected'});
   });
 
-  it('should throw error when `Create` button is clicked due to `Control is not locked`', async () => {
```

Comment:
Is the `Create` button not disabled by default anymore?
If it is maybe we can check that is disabled as not lock is taken

Reply:
I think it could be disabled until at least 1 detector is not selected, but I don't see any relation with the locks.